src/Presentation/RestApi/Controller/v1/SdkInitProjectController.php

```diff
@@ -22,38 +20,38 @@ class SdkInitProjectController
      *
      * @OA\RequestBody(
      *
-     *      @OA\JsonContent(
+     * @OA\JsonContent(
```

Comment:
Please revert these formatting changes if possible to avoid future merge conflicts. As you know we will remove it later